### ISSUE TITLE
Secure post-login cookie with SameSite attribute

### ISSUE THREAD

**Issue by mena-bosch** [eclipse/vorto#2615]
Noticed Firefox (81.1) is warning about the lack of SameSite attribute for the cookie used to redirect requests after login. 
This could mean the cookie will cease to function as is in the near or far future. 
Worth investigating if this can be secured proactively.

**Comment by mena-bosch** [comment]
Afterthought: this should be re-scoped to all cookies the application manipulates.

**Comment by mena-bosch** [comment]
After-afterthought: there are 3 cookies impacted. 

- The re-direct cookie was easy to fix programmatically. 

- The "cookie consent" cookie is loaded through a 3rd party npm library [cookieconsent](https://www.jsdelivr.com/package/npm/cookieconsent?path=build), delivered through [https://cdn.jsdelivr.net](https://cdn.jsdelivr.net). 
I have upgraded the latter to its later version (3.1.1), but it doesn't seem to set `SameContent` to anything by default as of yet, which means the value inferred is "None", which in turn means it will someday be a problem with recent browsers. 
For now, the only solution I can think of is to monitor the library for changes, and upgrade again if there's any good news. 
If, instead, the library does not provide a relevant update in due time, we may have to consider changing this mechanism. 

- The JSESSIONID. This one is the most troublesome. The _only_ solution I haven't tried yet was to use a configuration bean setting up a `org.springframework.session.web.http.DefaultCookieSerializer` - this is because the [setSameSite](https://docs.spring.io/spring-session/docs/current/api/org/springframework/session/web/http/DefaultCookieSerializer.html#setSameSite-java.lang.String-) method is not available for our current Spring Boot base version (1.5.13). 
All other solutions I could find (servlet filter, authentication listener) failed to work or were inapplicable (tomcat config).
Upgrading to Spring Boot 2 for the only purpose of testing this last solution is way out of scope here, so it will need to wait for a more dedicated task (and suitable time, given the effort required).  

**TL;DR**
This is going to fix the post-login redirect cookie _only_, and upgrade the cookieconsent library to its latest version as an extraneous "improvement".
